src/screens/SUE/SueListScreen.tsx

```diff
@@ -106,12 +96,11 @@ export const SueListScreen = ({ navigation, route }: Props) => {
   const query = route.params?.query ?? '';
 
   const limit = 20;
-  const initial_start_date = '1900-01-01T00:00:00+01:00';
-  const dataCountQueryVariables = {
-    start_date: initial_start_date
-  };
+  const initial_start_date = { start_date: '1900-01-01T00:00:00+01:00' };
+  const [dataCountQueryVariables, setDataCountQueryVariables] = useState({
+    initial_start_date
+  });
```

Comment:
there will be an key `initial_start_date` this way, but i guess you want `start_date` in here.
so it needs to be `useState(initial_start_date)`

Reply:
yes you are right. I will update.